message/pool/message.go

```diff
@@ -135,11 +135,20 @@ func (r *Message) SetCode(code codes.Code) {
 	r.isModified = true
 }
 
-func (r *Message) SetETag(value []byte) {
+// SetETag insert/update ETag option.
+//
+// Option definition:
+// 	- format: opaque, length: 1-8, repeatable
```

Comment:
@jkralik apparently, ETag should be repeatable. I guess we should add a `AddETag` function and the setter should remove all existing ETag options before calling `SetOptionBytes`.

Reply:
Ok. pls create `AddETag`

Reply:
Finally created! :D